### PR TITLE
Minor CH32 documentation additions

### PR DESCRIPTION
**Pull request by cw2**
I apologize in advance for not creating a dedicated issue, hope this helps 😊

### CODE REVIEW

File: docs/CH32V30x.md [hunk 1]
```diff
@@ -1,8 +1,28 @@
 # CH32V30x
 
-- CH32V303
-- CH32V305
-- CH32V307
+- CH32V303xBT6: RISC-V4F, 128KB Flash, 32KB SRAM, 144MHz, USB2.0 FS
```
Comment (andelf) [inline docs/CH32V30x.md:3]: Wrong description.
All CH32V30x has 480K Flash, only "zero-wait" Flash section varies.

File: docs/CH32V20x.md [hunk 2]
```diff
@@ -1,15 +1,32 @@
 # CH32V20x
 
-- PA13 SWDIO
-- PA14 SWCLK
+- CH32V203: RISC-V4B, 64KB Flash, 20KB SRAM, 144MHz, USB2.0
+- CH32V208: RISC-V4C, 128KB Flash, 64KB SRAM, 144MHz, USB2.0, ETH10M
```
Comment (andelf) [inline docs/CH32V20x.md:4]: All V203 series have a Flash size of 224K.
V208 has a Flash size of 480K.

File: docs/CH32V30x.md (hunk 1 above)
Comment (cw2) [reply docs/CH32V30x.md:3]: Thanks for the correction, I have already noticed it in the main Readme. I'll push a fix later today (my timezone).

**Comment by cw2** [comment]
I have updated the PR with memory size description fix.

**Comment by cw2** [comment]
Closed due to lack of interest.

**Pull request closed without merging**